src/test/resources/api/dummy_restipiexample/_post/rqpost.json

```diff
@@ -0,0 +1 @@
+{
  "data": {
    "name": "test",
    "salary": "123",
    "age": "23"
  }
}
```

Comment:
lets generate dynamically name, salary and age for each test. Check doc and demo tests